### PR TITLE
Inject BF version

### PR DESCRIPTION
**Pull request by jburel**
Set the version of BF
since will then be used in the doc without other change. Only rolling back a commit in autogen

### CODE REVIEW

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-build-push#114](https://web-proxy.openmicroscopy.org/west-ci/job/OMERO-build-push/114/). See the [console output](https://web-proxy.openmicroscopy.org/west-ci/job/OMERO-build-push/114/consoleText) for more details.
Possible conflicts:
  - PR #28 jburel 'BF 6.1.0-m1'
    - build.gradle

--conflicts

**Comment by sbesson** [comment]
This was not included into today's CI build. I understand the usage of the properties file to introspect the Bio-Formats version down the application stack. We might be careful about wording so that this does not become interpreted as being able to set the Bio-Formats version from a server configuration.

Primary question from my side is whether this strategy is compatible with the ongoing work to try and re-couple Bio-Formats SNAPSHOT with OMERO 5.5 SNAPSHOT. So far, the idea was to use the same approach as the one developed for coupling OMERO components i.e.:

- let `bio-formats-build` generate a properties file (see https://github.com/ome/build-infra/pull/7)
- update [gradle-set-dependencies](https://github.com/ome/build-infra/blob/master/gradle-set-dependencies) to consume this file and update `ome:*` dependencies.

Would this change would affect this workflow?

**Comment by jburel** [comment]
As long as the properties file is then accessible and we can parse it in CLI then I do not see why not
This is the "quickest" solution at this point in time i.e. the one requiring less work.

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-build-push#115](https://web-proxy.openmicroscopy.org/west-ci/job/OMERO-build-push/115/). See the [console output](https://web-proxy.openmicroscopy.org/west-ci/job/OMERO-build-push/115/consoleText) for more details.
Possible conflicts:
  - PR #28 jburel 'BF 6.1.0-m1'
    - build.gradle

--conflicts

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-build-push#116](https://web-proxy.openmicroscopy.org/west-ci/job/OMERO-build-push/116/). See the [console output](https://web-proxy.openmicroscopy.org/west-ci/job/OMERO-build-push/116/consoleText) for more details.
Possible conflicts:
  - PR #28 jburel 'BF 6.1.0-m1'
    - build.gradle

--conflicts

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [OMERO-build-push#117](https://web-proxy.openmicroscopy.org/west-ci/job/OMERO-build-push/117/). See the [console output](https://web-proxy.openmicroscopy.org/west-ci/job/OMERO-build-push/117/consoleText) for more details.
Possible conflicts:
  - PR #28 jburel 'BF 6.1.0-m1'
    - build.gradle

--conflicts

**Comment by jburel** [comment]
Closing this PR for now

File: build.gradle [hunk 1]
```diff
@@ -32,7 +33,7 @@ dependencies {
     api("org.hibernate:hibernate-search:3.4.2.Final")
 
     // Our libraries
-    api("ome:formats-gpl:5.9.2") {
```
Comment (joshmoore) [inline build.gradle]: This is going to cause an issue for build-infra. We'll need a PR on that repository in sync with this PR to keep things building.

Comment (joshmoore) [reply build.gradle]: Using `gpl:${bf_version}") { ` _might_ work.

**Pull request closed without merging**